app/views/users/show.erb

```diff
@@ -8,7 +8,30 @@
     <a href='/users/<%="#{@user.id}"%>/edit'>Update Profile </a> <br><br>
     <a href= '/users/<%="#{@user.id}"%>/edit'>Delete Account </a>
   </div>
-  <section>
-    <h1> Game Record </h1>
-  </section>
+  <br><br>
+  <fieldset>
+    <legend> <em><b>Game Record</b></em></legend>
+    <div>
+      <label>Total games: <%= @user.games.count%></label><br>
+      <label>Average Score: <%= @user.average_score%></label>
+      <% if @user.games %>
+        <h3> List of topics with score </h3>
+        <% count = 1 %>
+        <!-- User.first.games.first.guesses.first.topic.topic_name -->
+        <% list = @user.games %>
+        <% list.each do |game| %>
+            <% if game.guesses %>
+              <p>Game <%= count%>: <%= game.guesses.first.topic.topic_name %>  <%= game.score %></p>
```

Comment:
undefined method `topic' for nil:NilClass

I got this error right when I logged in from my account?